main/lua5.4-lpeg/patches/makefile.patch

```diff
@@ -0,0 +1,13 @@
+use cbuild compiler flags
```

Comment:
the patches directory in lua5.4-lpeg should be a symlink to the one in lua5.1-lpeg instead